src/app/header/header.component.html

```diff
@@ -189,8 +191,16 @@ <h5 id="offcanvasStartLabel">penda</h5>
                         />
                         <span class="dark-mode-slider"></span>
                     </label>
-                </div>
+                </div>        
             </div>
+            <ng-template 
+            [ngIf]="darkModeService.isDarkMode" 
+            [ngIfElse]="textToChange">
```

Comment:
`textToChange` could be used anywhere else! The name should be more meaningful, e.g. something like `otherModeText`.